#tugas


untuk soal dibawah ini temukan asimtot 2 berada pada n berapa?

1+1/2+1/3+⋯+1/n<2

In [3]:
limit = 2
total = 0

for n in range(1, 10):
    total += 1/n
    if total >= limit:
        print(f"n pertama kali melewati {limit} adalah n={n}, jumlah = {total}")
        break


n pertama kali melewati 2 adalah n=4, jumlah = 2.083333333333333


Pada soal ini diminta mencari nilai n pertama kali sehingga jumlah deret tersebut melampaui angka 2. Jika dihitung secara manual, untuk n=1 jumlahnya 1,
untuk n=2 jumlahnya 1.5, untuk n=3 jumlahnya 1.833..., dan pada n=4 jumlahnya menjadi 2.083... yang sudah lebih besar dari 2. Dengan demikian, n=4 adalah nilai pertama kali deret harmonik melampaui angka 2.